pkg/server/commands/reverseexpand/pipeline.go

```diff
@@ -1320,7 +1328,7 @@ func (s *sender) edge() *Edge {
 }
 
 func (s *sender) loop(fn loopFunc) {
-	for msg := range s.prod.seq() {
+	for msg := range s.prod.seq(context.Background()) {
```

Comment:
Q: Do we need to use a background context + some reasonable deadline (10 minutes? 30 minutes?)

Otherwise, this means that there is possibility that the seq can hang.